src/pages/Sketch/components/Canvas.jsx

```diff
@@ -449,7 +452,7 @@ function Canvas() {
         types: [
           {
             description: "Doc2pen Sketch Save File",
-            accept: { "application/d2ps": [".d2ps"] },
+            accept: { "image/*": [".jpg .jpeg .png"] },
```

Comment:
@anshika208 Why did you delete these lines. You are breaking an existing feature

Moreover, this function is to "Save" file. It's not for "loading" file. You edited the wrong function.